Configurations/release-move-tag.sh

```diff
@@ -10,17 +10,17 @@ if [ "$commits_since_tag" -gt 0 ]; then
     echo "Package.swift has not been committed and tag has not been moved."
 else
     # TODO: add sanity check to see if version is actually being updated or not?
-    read -p "Do you want to commit changes to Package.swift and force move tag '$latest_git_tag'? (required for Swift Package Manager release) " -n 1 -r
+    read -p "Do you want to commit changes to Package.swift and force move tag '$latest_git_tag'? (required for SPM release) [Y/n]" -n 1 -r
     echo    # (optional) move to a new line
-    if [[ $REPLY =~ ^[Yy]$ ]]; then
+    if [[ $REPLY =~ ^[Nn]$ ]]; then
```

Comment:
The prompt now defaults to YES. Since this is only ever relevant when you've literally just tagged a new release and run `make release` afterwards, should the prompt be skipped entirely? In other words, should Package.swift *always* be updated, committed and the tag moved in those circumstances?

Reply:
Prompt is fine